lottie-swift/src/Public/iOS/AnimationFromAssetLoading.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  AnimationFromAssetLoading.swift
+//  Lottie
+//
+//  Created by Fouquet, René on 22.07.19.
+//
+
+import Foundation
+import UIKit
+
+public extension AnimationView {
```

Comment:
Would be nice if this solution supported MacOS as well.

These extensions could be added to `AnimationPublic.Swift` and `AnimationViewInitializers.swift` with an additional wrapper added around the Asset catalog stuff.